test/e2e/alphaupdate/update_test.go

```diff
@@ -83,18 +96,16 @@ var _ = Describe("kubebuilder", func() {
 				}
 			}
 
-			// Clean up TestContext
-			if kbc != nil {
-				kbc.Destroy()
-			}
+			_ = os.RemoveAll(kbc.Dir)
```

Comment:
We should always remove the directory
If we call Destroy it will remove the image 
@vitorfloriano